### ISSUE TITLE
Use external terminal / collab with microsoft/terminal

### ISSUE THREAD

**Issue by Morgy93** [microsoft/vscode-remote-release#4703]
Hello,

maybe this is related to #3413 but since I don't want to use the debugger or any other "extension", I'm not sure.

However, I'd love to use an external terminal with remote ssh which seems not possible right now and maybe it's not easy to achieve in general.
So I thought that the most logical way would be to collaborate with https://github.com/Microsoft/Terminal wouldn't it?
If I could use the Windows Terminal just like the integrated terminal it would solve all my issues with integrated terminal in general and working remote ssh.

Would love to hear that there is any chance of achieving a collaboration. 😊

**Comment by roblourens** [comment]
It's not clear what you are asking for. What do you want to do with an external terminal and remote ssh? Please give some more details

**Comment by Morgy93** [comment]
Well, an external terminal (Windows Terminal in my case) solves two of my biggest issues with the integrated terminal right away:
- https://github.com/microsoft/vscode/issues/10546
- https://github.com/microsoft/vscode/issues/10121 (and with it https://github.com/microsoft/vscode/issues/26777 in a way)

VSCode does support an external terminal, just not via this official remote ssh extension. (Could not find a documentation, but it's mentioned here https://code.visualstudio.com/docs/getstarted/settings#_default-settings)
I think that's basically a lack of consistency of features, since it's an official extension.

Basically, I'd love to work with an external terminal on remote ssh just like with the integrated one.
So that you can right click in explorer and "Open in External Terminal" and even if there's a path written in the console that you can `CTRL+Left Mouse Click` on it to open it in VSCode right away - just like it does on the integrated terminal.
I'd understand if the latter is difficult to implement, but that's why I suggested the collaboration with https://github.com/Microsoft/Terminal which might be easier to implement (since both is from Microsoft).

**Comment by Tyriar** [comment]
I created an extension that provides good integration with Windows Terminal, it also supports remote SSH if the extension is installed which I think is exactly what you're after: https://marketplace.visualstudio.com/items?itemName=Tyriar.windows-terminal
